dds/DCPS/transport/multicast/MulticastSession.h

```diff
@@ -92,12 +92,12 @@ class OpenDDS_Multicast_Export MulticastSession
 
   virtual void syn_hook(const SequenceNumber& /*seq*/) {}
 
-  typedef ACE_Reverse_Lock<ACE_Thread_Mutex> Reverse_Lock_t;
-  Reverse_Lock_t reverse_start_lock_;
-
   ACE_Thread_Mutex start_lock_;
   bool started_;
 
+  typedef ACE_Reverse_Lock<ACE_Thread_Mutex> Reverse_Lock_t;
+  Reverse_Lock_t reverse_start_lock_;
```

Comment:
Looks like there is a reorder warning from the initialization of `reverse_start_lock_` and `started`.